openrtb-core/src/main/protobuf/openrtb-ext.proto

```diff
@@ -287,10 +351,33 @@ message SKAdNetworkResponse {
   extensions 100 to 9999;
 }
 
+message DsaResponse {
+  // Advertiser Transparency: Free UNICODE text string with a name of whose
+  // behalf the ad is displayed. Maximum 100 characters.
+  optional string behalf = 1;
+
+  // Advertiser Transparency: Free UNICODE text string of who paid for the ad.
+  // Must always be included even if it's the same as what is listed in the
+  // behalf attribute. Maximum 100 characters.
+  optional string paid = 2;
+
+  // Array of objects of the entities that applied user parameters and the
+  // parameters they applied.
+  repeated Transparency transparency = 3;
```

Comment:
Could you please confirm that transparency field will repeated(not optional)?
I have small doubts because I see a discrepancy in [document](https://github.com/InteractiveAdvertisingBureau/openrtb/blob/main/extensions/community_extensions/dsa_transparency.md#:~:text=Advertiser%22%2C%0A%20%20%20%09%09%09%09%20%22paid%22%3A%20%22Advertiser%22%2C%0A%20%20%20%09%09%09%09%20%22-,transparency,-%22%3A%20%7B%0A%20%20%20%09%09%09%09%09%20%22domain%22%3A%20%E2%80%9Cdsp1domain.com)

Reply:
A `repeated` field is inherently optional -- the array can be empty. Per the spec, this field is an array of objects, so `repeated` is the appropriate type. DM me if you have other questions here.